scripts/check_configs.ts

```diff
@@ -210,6 +217,50 @@ function isValidAdditionalIbcTokenConfig(addTokenConfigArr: AdditionalIbcTokenCo
   return true;
 }
 
+function isValidDemexTradingLeagueConfig(
+  demexTradingLeagueConfig: DemexTradingLeagueConfig, 
+  network: CarbonSDK.Network, 
+  marketIds: string[],
+  blacklistedMarkets: string[],
+  perpPoolIds: string[],
+  tokenSymbols: string[]) {
+  
+  const hasInvalidPromoMarkets = checkValidEntries(demexTradingLeagueConfig.promoMarkets, marketIds);
+  if (hasInvalidPromoMarkets.status && hasInvalidPromoMarkets.entry) {
+    let listOfInvalidMarkets: string = hasInvalidPromoMarkets.entry.join(', ');
+    console.error(`ERROR: ${network}.json has the following invalid promo market entries in demex_trading_league_config: ${listOfInvalidMarkets}. Please make sure to only input valid markets in ${network}`);
+    return false;
+  }
+
+  const hasDuplicatePromoMarkets = checkDuplicateEntries(demexTradingLeagueConfig.promoMarkets);
+  if (hasDuplicatePromoMarkets.status && hasDuplicatePromoMarkets.entry) {
+    let listOfDuplicates: string = hasDuplicatePromoMarkets.entry.join(", ");
+    console.error(`ERROR: ${network}.json has the following duplicated promo market entries in demex_trading_league_config: ${listOfDuplicates}. Please make sure to only input each market once in ${network}`);
+    return false;
+  }
+  
+  const hasBlacklistedMarketsInPromo = checkBlacklistedMarkets(demexTradingLeagueConfig.promoMarkets, blacklistedMarkets);
+  if (hasBlacklistedMarketsInPromo.status && hasBlacklistedMarketsInPromo.entry) {
+    let listOfBlacklistedMarkets: string = hasBlacklistedMarketsInPromo.entry.join(", ");
+    console.error(`ERROR: ${network}.json has the following blacklisted market entries in promo markets entries in demex_trading_league_config: ${listOfBlacklistedMarkets}. Please make sure that blacklisted markets are not found in promo markets in ${network}`);
+    return false;
+  }
+
+  const hasInvalidCurrentCompPerpPoolId = checkValidEntries([demexTradingLeagueConfig.currentCompPerpPoolId.toString()], perpPoolIds)
```

Comment:
Since there is only 1 item, you can do the direct check in the array instead:
```typescript
const hasInvalidCurrentCompPerpPoolId = perpPoolIds.includes(demexTradingLeagueConfig.currentCompPerpPoolId.toString());
```